Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -1546,15 +1546,16 @@ public void onError(final RequestError error) {
   }
 
   @Nullable
-  public SourceFileEdit edit_importElements(@NotNull final VirtualFile file, @NotNull final List<ImportedElements> importedElements) {
+  public SourceFileEdit edit_importElements(@NotNull final VirtualFile file, @NotNull final List<ImportedElements> importedElements,
+                                            final int offset) {
     final String filePath = FileUtil.toSystemDependentName(file.getPath());
     final Ref<SourceFileEdit> resultRef = new Ref<>();
 
     final AnalysisServer server = myServer;
     if (server == null || StringUtil.compareVersionNumbers(mySdkVersion, "1.25") < 0) return null;
 
     final CountDownLatch latch = new CountDownLatch(1);
-    server.edit_importElements(filePath, importedElements, new ImportElementsConsumer() {
+    server.edit_importElements(filePath, importedElements, offset, new ImportElementsConsumer() {
```

Comment:
I've wrapped the offset into `getOriginalOffset()` to make sure it works on Windows.